src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs

```diff
@@ -38,13 +38,15 @@ internal class WorkerFunctionMetadataProvider : IWorkerFunctionMetadataProvider
             _environment = environment;
             _channelManager = webHostRpcWorkerChannelManager;
             _workerRuntime = _environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionWorkerRuntime);
+            _logger.LogDebug("WorkerFunctionMetadataProvider initialized with workerRuntime:{workerRuntime}", _workerRuntime);
```

Comment:
```suggestion
            _logger.LogDebug("WorkerFunctionMetadataProvider initialized with workerRuntime: {workerRuntime}", _workerRuntime);
```

Reply:
Also for things like type/method names, use nameof so if types are renamed the logs aren't out of sync.